Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:2.7.8
```

Comment:
We have ubuntu based python image :eyes: 
https://quay.io/repository/wantedly/python